messages/en.json

```
@@ -16,5 +16,8 @@
   "store/product-list.originalPrice": "Original price",
   "store/product-list.changeTo": "Change to",
   "store/product-list.manualPricePlaceholder": "Type a monetary value",
-  "store/product-list.applyManualPrice": "ok"
+  "store/product-list.applyManualPrice": "ok",
+  "store/product-list.quantitySelector.label": "{quantity} {measurementUnit}",
+  "store/product-list.quantitySelector.maxValueLabel": "{quantity} {measurementUnit} +",
+  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by factions of {unitMultiplier}{measurementUnit}. Given that, the inserted quantity has been rounded to {roundedValue}{measurementUnit}."
```

We better not allow factions as our sellers :kekw:
```suggestion
  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by fractions of {unitMultiplier}{measurementUnit}. Given that, the inserted quantity has been rounded to {roundedValue}{measurementUnit}."
```